webapp/src/main/webapp/angular.json

```diff
@@ -118,8 +118,7 @@
           "options": {
             "tsConfig": [
               "tsconfig.app.json",
-              "tsconfig.spec.json",
-              "e2e/tsconfig.json"
+              "tsconfig.spec.json"
```

Comment:
Could you explain why is it necessary?

Reply:
Project does not contain e2e/tsconfig.json file